packs/feat-effects/effect-dash-of-herbs-immunity.json

```diff
@@ -0,0 +1,36 @@
+{
+    "_id": "ZTB5HEJJck1ROWAQ",
+    "img": "icons/magic/nature/leaf-elm-beam-green.webp",
+    "name": "Effect: Dash of Herbs Immunity",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Dash of Herbs]</p>\n<p>A creature <span style=\"font-family:Roboto, sans-serif\">becomes temporarily immune to Dash of Herbs for 10 minutes.</span></p>"
```

Comment:
Formatting snuck in. I would scrap this whole effect anyway. The system hasn't done purely tracking effects for a while.